presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Fix a bug where index joins would fail with the error ``driver should never block``.
+  Queries will now fail if the index is unable to load within the time specified by
+  the ``index-loader-timeout`` configuration property and the ``index_loader_timeout``
+  session property.
+* Fix a bug that could cause ``NullPointerException`` in spatial join with geometry
+  collections on the build side.
+* Improve performance for queries with ``FULL OUTER JOIN`` where join keys have the
+  ``COALESCE`` function applied.
+* Improve performance for ``UnnestOperator`` by using ``DictionaryBlock``.
+* Improve performance for ``PartitionedOutputOperator``. The new
+  ``OptimizedPartitionedOutputOperator`` can be enabled by the ``optimized_repartitioning``
+  session property or the ``experimental.optimized-repartitioning`` configuration property.
+* Add spatial join (broadcast and distributed) support for :func:`ST_Equals`,
```

Comment:
we've been referring to the two kinds of joins as "broadcast" and "partitioned", so do the same here for consistency.

Reply:
@rschlussel Thanks for the correction. Does the following look good?
```
* Add spatial join (``BROADCAST`` and ``PARTITIONED``) support for :func:`ST_Equals`,
  :func:`ST_Overlaps`, :func:`ST_Crosses`, and :func:`ST_Touches`.
```

Reply:
@rongrong already addressed the comments!